isis/tests/FunctionalTestsFits2isis.cpp

```diff
@@ -0,0 +1,139 @@
+#include <QTemporaryDir>
+
+#include "fits2isis.h"
+#include "Fixtures.h"
+#include "Pvl.h"
+#include "PvlGroup.h"
+#include "TestUtilities.h"
+#include "Histogram.h"
+
+#include "gtest/gtest.h"
+
+using namespace Isis;
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/fits2isis.xml").expanded();
+
+TEST(Fits2Isis, Fits2IsisTestDefault) {
+  QTemporaryDir prefix;
+  QString cubeFileName = prefix.path() + "/fits2isisTEMP.cub";
+  QVector<QString> args = {"from=data/fits2isis/default.fits", "to=" + cubeFileName };
+
+  UserInterface options(APP_XML, args);
+  try {
+   fits2isis(options);
+  }
+  catch (IException &e) {
+    FAIL() << "Unable to ingest fits file: " << e.toString().toStdString().c_str() << std::endl;
+  }
+  Cube cube(cubeFileName);
+  Pvl *isisLabel = cube.label();
+
+  // Dimensions Group
+  ASSERT_EQ(cube.sampleCount(), 200);
+  ASSERT_EQ(cube.lineCount(), 25);
+  ASSERT_EQ(cube.bandCount(), 3);
```

Comment:
Using the fits cropper, you should also be able to limit the sample count to 25. Should just be another indexer on the output data array.